.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,39 @@
+name: Build and test
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
```

Comment:
Is it possible to check several Python versions?

Reply:
Yep, it's now testing on 3.6-3.8.